src/index.html

```diff
@@ -20,8 +20,450 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo__image"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="/apple"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="/samsung"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="/smartphones"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="/laptops_computers"
```

Comment:
The `href` attribute for the 'laptops & computers' link contains a double underscore (`/laptops_computers`). Please verify if this is intentional. If not, consider correcting it to a single underscore or another appropriate character.